data/categories.txt

```diff
@@ -2126,7 +2126,7 @@ fr:Arrêt de bus|2bus|arrêt|2autobus
 de:Bushaltestelle|Haltestelle|2Bus|Autobus|ÖPNV
 hu:2Buszmegálló|2busz|megálló
 id:3Halte bus
-it:3Fermata|autobus
+it:3Fermata dell'autobus|autobus
```

Comment:
NOTE: I'm not 100% sure that I'm respecting the logic, but it's not just "Fermata" but "Fermata dell'autobus" if related to an autobus.

Reply:
Can the user type "fermata" in the search to find _any_ type of stop? If yes, then please add it also as a synonym here and below.

Reply:
Category search does match separate tokens always. This change will affect suggestion's name only. Search will work the same way for "fermata", "autobus", "tram".

The only thing that _entropy_ is increasing a bit with "del/dell" :)

Reply:
If I understand correctly, "fermata" and "autobus" are just tokens, so feel free to say "yeah please drop -dell-" since it just means "of the". Unless an exact string match is not is weight-rewarded.

Reply:
Did you test this changes somehow? "dell" is ok if we want to suggest grammatically valid suggest name.
Or "Fermata autobus", "Fermata tram" are also good suggestions?

Reply:
Yeah! Here all the variants that could be nice to take in consideration:

- fermata dell'autobus / fermata autobus
- fermata del pullman / fermata pullman
- fermata del bus / fermata bus
- bus stop